modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1489,6 +1489,103 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the user is able to add the Formula Field when managing Custom Validation"
+	@priority = 4
+	test CanManageFormulaFieldWithCustomValidation {
+		property custom.properties = "feature.flag.LPS-164948=true";
+
+		task ("Given: an Object with Formula Field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 176853",
+				objectName = "CustomObject176853",
+				pluralLabelName = "Custom Objects 176853");
+
+			for (var valueField : list "1,2") {
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Integer",
+					fieldLabelName = "Custom Integer ${valueField}",
+					fieldName = "customInteger${valueField}",
+					fieldType = "Integer",
+					isRequired = "false",
+					objectName = "CustomObject176853");
+			}
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Formula",
+				fieldLabelName = "Custom Formula Field",
+				fieldName = "customFormulaField",
+				fieldType = "String",
+				formulaScript = "customInteger1 + customInteger2",
+				isRequired = "false",
+				objectName = "CustomObject176853",
+				outputOption = "Integer");
+		}
+
+		task ("When: managing a Custom Validation") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 176853");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.goToValidationsTab();
+
+			ObjectCustomValidation.addObjectValidation(
+				validationName = "Custom Validation Formula Plus",
+				validationType = "Expression Builder");
+
+			ObjectCustomValidation.gotoCustomValidation(validationName = "Custom Validation Formula Plus");
+
+			ObjectCustomValidation.activeValidation();
+
+			ObjectCustomValidation.gotoTab(tabName = "Conditions");
+
+			ObjectCustomValidation.insertScriptOnValidation(scriptValidation = "customFormulaField <= 50");
+
+			ObjectCustomValidation.addErrorMessage(errorMessage = "Error not accepted greater than 50");
+
+			PortletEntry.save();
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject176853");
+		}
+
+		task ("Then: is possible to validate the Formula Field") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject176853");
+
+			LexiconEntry.gotoAdd();
+
+			ObjectPortlet.typeCustomField(
+				customField = "Custom Integer 1",
+				entry = 25);
+
+			ObjectPortlet.typeCustomField(
+				customField = "Custom Integer 2",
+				entry = 25);
+
+			PortletEntry.save();
+
+			Navigator.gotoBack();
+
+			ObjectPortlet.viewEntry(entry = 50);
+
+			LexiconEntry.gotoAdd();
+
+			ObjectPortlet.typeCustomField(
+				customField = "Custom Integer 1",
+				entry = 30);
+
+			ObjectPortlet.typeCustomField(
+				customField = "Custom Integer 2",
+				entry = 30);
+
+			Button.click(button = "Save");
+
+			AssertTextEquals.assertPartialText(
+				locator1 = "Message#ERROR",
+				value1 = "Error not accepted greater than 50");
```

Comment:
value1 = "The sum must be less than or equal to 50");

Reply:
Done !